### ISSUE TITLE
PhotoFlow translations

### ISSUE THREAD

**Issue by aferrero2707** [aferrero2707/PhotoFlow#100]
This is an enhancement issue to centralise all translation-related discussions.

**Comment by aferrero2707** [comment]
@Zbyma72age @Spirou4D I have updated the fr.po and pl.po files in the stable branch:
https://github.com/aferrero2707/PhotoFlow/tree/stable/po

The best way to update the translations would be to edit those files and then send me back the modified versions (or do a fork & pull request).

My idea would be to finalise the release of version 0.2.7 in two or three weeks from now.

Thanks!

**Comment by Spirou4D** [comment]
Ok Andrea, no problem! I begin tomorow....
At soon
Patrick D.

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2016-08-25 21:22 GMT+02:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
Hi Andrea,
French translation made! But there are yet a lot of words to translate in PhotoFlow....

Tryed on my PhotoFlow development branch v0.2.6-pre

**Comment by Zbyma72age** [comment]
Hi
I prepared new a translation into Polish photoflow.po and photoflow.mo,
Through Google Drive

https://drive.google.com/open?id=0By0VZc42GQRFY01Qd1JaQ05PazA
Zbyma72age


2016-08-25 21:22 GMT+02:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
@Spirou4D: where can I find the updated French translations .po file?
Thanks!

@Zbyma72age: I've pushed your file to the stable branch on github, and it will be included in next release.
Thanks!

**Comment by Spirou4D** [comment]
2016-08-31 10:09 GMT+02:00 aferrero2707 <notifications@github.com>:

> @Spirou4D <https://github.com/Spirou4D>: where can I find the updated
> French translations .po file?
> Thanks!
>
>
I have open your github .po page and paste my translation simply.

Byebye
Patrick

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

**Comment by aferrero2707** [comment]
Could you also send me the fr.po file by email, so that I can verify that the github version has been updated properly?

Thanks!

**Comment by Spirou4D** [comment]
Ok, now.

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2016-08-31 11:02 GMT+02:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
Thanks!

**Comment by Spirou4D** [comment]
With pleasure, Andrea. there are yet a lot of work.....

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2016-08-31 11:12 GMT+02:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
French translations committed to the stable branch as well.

**Comment by Spirou4D** [comment]
Very good Andrea!

But i have  a question: Why there are yet a lot of words not translated,
please?

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2016-08-31 15:58 GMT+02:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
The words to be translated need to be explicitly tagged in the source code... and I simply still need to tag many text strings. 
It's work in progress, but it had a quite low priority until now. 

I plan some major changes after 0.2.7 will be out, and at that point I will also complete the tagging of text strings that need translation.

**Comment by Spirou4D** [comment]
Ha ok I understand now.
Good Luck Andrea!
At soon
Patrick

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2016-08-31 16:10 GMT+02:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
Hi Carmelo,

I write you directly because i don't receive answer by google+ Photoflow
account.
How are you, my Friend?

Have you seen: Partha has embedded Nufraw in his Win Gimp compilation?

Do you work on a Photoflow embedded plugin for Krita, please? Have you seen
that David Revoy said:
"Dmitry's work on supporting SVG and improving the vector tools is going
really well (in Krita)."

There already are a G'Mic plugin for Krita, the must would be a Photoflow
plugin, i mean, not?

I hope your health is good!
I wishes you full of Luck!
Patrick


---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

**Comment by aferrero2707** [comment]
Hi Patrick!

All is fine for me, I'm just quite busy and therefore silently working on the code without much time to write about my progresses on social media... anyhow, thanks for asking! What about you?

The switch to nUFRaw by Partha was somehow expected and discussed, as I have been busy re-writing the PhF plugin interface in the past few weeks.

Now the new code is ready, and it should be much easier now to include the PhF plug-in in other projects, like Krita... more news in the near future!

**Comment by Spirou4D** [comment]
2017-02-19 12:43 GMT+01:00 aferrero2707 <notifications@github.com>:

> Now the new code is ready,


Good news Carmelo! I just use it with the new release of GIMP 2.75 DEV but
my old compilation don't run now, I must re-git your code and re-compile a
new one! If you want, i can make a try of the compilation process for every
body.....on Discuss forum tomorrow...
Invite me if you want, i am still at home.
Good sunday.
At soon...
Bye
Patrick
---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

**Comment by aferrero2707** [comment]
The gimp plug-in is now a separate project. you can find it [here](https://github.com/aferrero2707/PhFGimp), with some building instructions.

You also need to update photoflow to the latest GIT version, so that it can correctly interact with the plug-in.

Of course, I'm interested in any sort of feedback... thanks!

**Comment by Spirou4D** [comment]
Hi Carmelo,

I have follow the instruction but for:
$ cmake -DBABL_FLIPS_DISABLED=OFF -DCMAKE_BUILD_TYPE=Release ..

-- Checking for module 'lcms2'
--   No package 'lcms2' found
CMake Error at /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:367
(message):
  A required package was not found

But is installed on my linux Mint Serena 18.1 (ubuntu 16.04 Xenial)
[image: Images intégrées 1]

How continue, please?



---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 18:51 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
Here is the log file of this error in joined file, please:

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 18:51 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
*lcms2 error corrected installating liblcms2-dev but now a new one:*
--   Found lcms2, version 2.7
CMake Error at
/usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:148
(message):
  *Could NOT find TIFF (missing: TIFF_LIBRARY TIFF_INCLUDE_DIR)*
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPackageHandleStandardArgs.cmake:388
(_FPHSA_FAILURE_MESSAGE)
  /usr/share/cmake-3.5/Modules/FindTIFF.cmake:82
(FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  CMakeLists.txt:15 (find_package)

No idea, I stop.

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 18:51 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
Do you have installed the libtiff5-dev? In general, for all the missing dependencies you need to install the corresponding -dev package...

**Comment by Spirou4D** [comment]
Yes now yes and i have installed module 'gexiv2' too *but i have this
traceback now:*
patrinux@HOME01 ~/Applications/PhFGimp/build $ cmake
-DBABL_FLIPS_DISABLED=OFF -DCMAKE_BUILD_TYPE=Release ..
-- Checking for module 'gexiv2'
--   Found gexiv2, version 0.10.3
-- GEXIV2_INCLUDE_DIRS:
/usr/include/glib-2.0/usr/lib/x86_64-linux-gnu/glib-2.0/include
-- GEXIV2_LIBRARY_DIRS:
-- Checking for module 'gimp-2.0>=2.9.0'
--
CMake Error at /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:367
(message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:532
(_pkg_check_modules_internal)
  CMakeLists.txt:22 (pkg_check_modules)


I have open this file line  22:
>>pkg_check_modules(GIMP REQUIRED gimp-2.0>=2.9.0)
if (GIMP_FOUND)
    pkg_check_modules(GIMP_2_9 gimp-2.0>=2.9.0)
    if (GIMP_2_9_FOUND)
pkg_check_modules(GIMPUI gimpui-2.0>=2.9.0)
   if (GIMPUI_FOUND)
        set (HAVE_GIMP_2_9 "1")
   endif (GIMPUI_FOUND)
    else (GIMP_2_9_FOUND)
        set (HAVE_GIMP_2_9 "0")
    endif (GIMP_2_9_FOUND)
    MESSAGE( STATUS "GIMP_2_9_CFLAGS:         " ${GIMP_2_9_CFLAGS} )
    MESSAGE( STATUS "GIMP_2_9_INCLUDE_DIRS:         "
${GIMP_2_9_INCLUDE_DIRS} )
    MESSAGE( STATUS "GIMP_2_9_LIBRARY_DIRS:         "
${GIMP_2_9_LIBRARY_DIRS} )
    MESSAGE( STATUS "GIMP_2_9_LDFLAGS:         " ${GIMP_2_9_LDFLAGS} )
endif (GIMP_FOUND)

*But GIMP 2.95 DEV is installed!*

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:11 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
Could you check if you have a ```gimp-2.0.pc``` file somewhere under /usr?

You can run

```find /usr -name gimp-2.0.pc```

**Comment by Spirou4D** [comment]
patrinux@HOME01 ~/Applications/PhFGimp/build $ find /usr -name gimp-2.0.pc
patrinux@HOME01 ~/Applications/PhFGimp/build $
*NOTHING FOUND!*


---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:28 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
Then it seems that you do not have the GIMP development package installed... probably only the standard GIMP package?

**Comment by Spirou4D** [comment]
[image: Images intégrées 1]

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:41 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
I go to sleep now...
Later if you find something...

Good night Carmelo...
Byebye
Pat

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:41 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
EUREKA Andrea,

This night my gimp release has been updated and now :
patrinux@HOME01 ~ $ find /usr -name gimp-2.0.pc
/usr/lib/pkgconfig/gimp-2.0.pc

But I have always the same message in terminal....
I must re-open my computer to see....

At soon
Pat

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:41 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
2017-02-19 22:28 GMT+01:00 aferrero2707 <notifications@github.com>:

> find /usr -name gimp-2.0.pc


*Yes ok now this file exist with the new update of this night!*
But I have always the same error:
patrinux@HOME01 ~/Applications/PhFGimp/build $ cmake
-DBABL_FLIPS_DISABLED=OFF -DCMAKE_BUILD_TYPE=Release ..
-- GEXIV2_INCLUDE_DIRS:
/usr/include/glib-2.0/usr/lib/x86_64-linux-gnu/glib-2.0/include
-- GEXIV2_LIBRARY_DIRS:
-- Checking for module 'gimp-2.0>=2.9.0'
--
CMake Error at /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:367
(message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:532
(_pkg_check_modules_internal)
  CMakeLists.txt:22 (pkg_check_modules)



---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

**Comment by aferrero2707** [comment]
What is the output of this command?

    ```cat /usr/lib/pkgconfig/gimp-2.0.pc```

**Comment by Spirou4D** [comment]
>>pkg_check_modules(GIMP REQUIRED gimp-2.0>=2.9.0)
if (GIMP_FOUND)
    pkg_check_modules(GIMP_2_9 gimp-2.0>=2.9.0)
    if (GIMP_2_9_FOUND)
pkg_check_modules(GIMPUI gimpui-2.0>=2.9.0)
    if (GIMPUI_FOUND)
         set (HAVE_GIMP_2_9 "1")
    endif (GIMPUI_FOUND)
    else (GIMP_2_9_FOUND)
        set (HAVE_GIMP_2_9 "0")
    endif (GIMP_2_9_FOUND)
endif (GIMP_FOUND)

/usr/lib/pkgconfig/gimp-2.0.pc

Then why pkg_check_modules(GIMP REQUIRED gimp-2.0>=2.9.0)  don't run?


---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:28 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
patrinux@HOME01 ~ $ cat /usr/lib/pkgconfig/gimp-2.0.pc
prefix=/usr
exec_prefix=${prefix}
libdir=${exec_prefix}/lib
includedir=${prefix}/include
datarootdir=${prefix}/share

gimpdatadir=${datarootdir}/gimp/2.0
gimplibdir=${exec_prefix}/lib/gimp/2.0
gimpsysconfdir=/etc/gimp/2.0
gimplocaledir=${datarootdir}/locale

Name: GIMP
Description: GIMP Library
Version: 2.9.5
Requires: gdk-pixbuf-2.0 >= 2.30.7 cairo >= 1.12.2 gegl-0.3 >= 0.3.12
Libs: -L${libdir} -lgimp-2.0 -lgimpmath-2.0  -lgimpconfig-2.0
-lgimpcolor-2.0 -lgimpbase-2.0
Cflags: -I${includedir}/gimp-2.0
patrinux@HOME01 ~ $


---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-20 12:23 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
Andrea, installed packages:

[image: Images intégrées 1]



---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-19 22:41 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
> Then why pkg_check_modules(GIMP REQUIRED gimp-2.0>=2.9.0)  don't run?

I also don't understand at all... and that's the first time I see that GIMP is not detected...

**Comment by Spirou4D** [comment]
I installed this module in plus:
[image: Images intégrées 1]

No change but i send you the log file in joined zip

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-20 12:39 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
Here is all module for cmake on Mint Serena 64B:

[image: Images intégrées 1]

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-20 12:39 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
I cannot see attachments here... maybe you can send me a private email?

**Comment by Spirou4D** [comment]
I sended you to photoflow@gmail.com

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-20 13:00 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by Spirou4D** [comment]
Non here PhotoFlow Editor <photofloweditor@gmail.com>

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-02-20 13:00 GMT+01:00 aferrero2707 <notifications@github.com>:

**Comment by aferrero2707** [comment]
@Zbyma72age @Spirou4D I have updated the fr.po and pl.po files in the stable branch:
https://github.com/aferrero2707/PhotoFlow/tree/stable/po

The best way to update the translations would be to edit those files and then send me back the modified versions (or do a fork & pull request).

I am planning to finalise the release of version 0.2.8 in two or three weeks from now...

Thanks!

**Comment by Spirou4D** [comment]
Ok I make the translation....

---

Patrick DepoixArchitecte D.P.L.G.Historien-chercheur en histoire de
l'ArchitectureEx-Maître de conférence de l'Ecole d'Architecture de
Lille-Région NordSitArchi-ReaderNewS
<http://www.google.com/reader/shared/user%2F04161245489718164963%2Fstate%2Fcom.google%2Fstarred>

http://www.linkedin.com/pub/patrick-depoix/40/b3b/542136, rue d'Esquermes59000
LilleFranceTel   (+33/0) 952 51 33 25Port  (+33/0) 06 23 13 73 71
Skype:oikakeruPour répondre --> Mailto:patrick.depoix@gmail.com
<http://---%20http://www.linkedin.com/pub/patrick-depoix/40/b3b/542%20Patrick%20Depoix%20Architecte%20D.P.L.G.%20Historien-chercheur%20en%20histoire%20de%20l%27Architecture%20Ex-Ma%C3%AEtre%20de%20conf%C3%A9rence%20de%20l%27Ecole%20d%27Architecture%20de%20Lille-R%C3%A9gion%20Nord136,%20rue%20d%27Esquermes%2059000%20Lille%20France%20Tel%20%20%20%28+33/0%29%20952%2051%2033%2025%20Port%20%20%28+33/0%29%2006%2023%2013%2073%2071%20Fax%20%20%28+33/0%29%20826%2070%2004%2040%20Skype:oikakeru%20%20Pour%20r%C3%A9pondre%20--%3E%20Mailto:patrick.depoix@free.fr>

2017-03-10 23:10 GMT+01:00 aferrero2707 <notifications@github.com>:
